tools/iotjs_build_info.js

```diff
@@ -1,31 +0,0 @@
-/* Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
```

Comment:
This file should not be removed because [testrunner.py:156](https://github.com/Samsung/iotjs/blob/master/tools/testrunner.py#L156) uses that to determine the built-in modules.